src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp/Project/AbstractOmniSharpProjectSnapshotManagerChangeTrigger.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project;
+
+public abstract class AbstractOmniSharpProjectSnapshotManagerChangeTrigger
```

Comment:
Seems like you converted this from an interface to an abstract class, and that had some flow on effects in other things. Is it simpler to revert this back to an interface now?

Reply:
It is!